Shared/Sources/HomeSectionItem.swift

```diff
@@ -0,0 +1,10 @@
+// Copyright © 2020 Itty Bitty Apps. All rights reserved.
+
+import Foundation
+
+struct HomeSectionItem: SectionItem {
+  typealias item = HomeItem
```

Comment:
Type names should generally always start with an uppercase letter. This includes type aliases 👍 

Can you safely remove the explicit type alias declaration here, and let Swift's type inference fill in the holes, or is it still required?

Reply:
Yup, can be removed 👍